src/js/components/Select/doc.js

```diff
@@ -161,3 +161,53 @@ export const doc = Select => {
 
   return DocumentedSelect;
 };
+
+export const themeDoc = {
+  'select.background': {
+    description: 'The background color used for Select.',
+    type: 'string',
+    defaultValue: undefined,
+  },
+  'select.container': {
```

Comment:
I don't think we need to document this intermediate key. `select.container.extend` should be sufficient.